.travis.yml

```diff
@@ -1,3 +1,3 @@
 language: node_js
 node_js:
-- 4
\ No newline at end of file
+- 8
```

Comment:
8.9? because hapi 17 is compatible with 8.9 and above https://github.com/hapijs/hapi